bpf/bpf_lxc.c

```diff
@@ -535,12 +535,12 @@ int tail_handle_ipv6(struct __ctx_buff *ctx)
 					false);
 			if (IS_ERR(ret))
 				return ret;
-
-			/* Store state to be picked up on the continuation tail call. */
-			lb_ctx_store_state(ctx, &ct_state_new);
```

Comment:
Shouldn't commit `datapath: Be more robust about state between tail calls` be merged with the previous commit? As far as I can see, it only touches lines introduces by the previous commit.

Reply:
Good point, squashed :-)